04.ls/ls.rb

```diff
@@ -98,8 +91,24 @@ def fetch_file_details(filenames)
   end
 end
 
-def find_max_length(key, file_details)
-  file_details.map { |file_detail| file_detail[key].to_s.length }.max
+def find_max_length(file_details)
+  %i[nlink user_name group_name file_size].to_h do |key|
+    file_detail_display_length = file_details.map do |file_detail|
```

Comment:
```suggestion
    values = file_details.map do |file_detail|
```

寿命が短い変数なので、シンプルな名前を付けても問題ありません